{"error": null, "issues": [{"address": 2452, "code": "t(_stage, msg.sender", "debug": "timestamp: 0xa8795cd3fffffffffffffffffffffffffffffffffffffffffffffffffffffffe\nstorage_6: 0x57f25cd000000000000000000000000000000000000000000000000000000000\nstorage_0 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\ncalldatasize_Redline: 0x3\nstorage_keccac_0: 0x0\nstorage_78541660797044910968829902406342334108369226379826116161446442989268089806461: 0x0\ncallvalue: 0x0\n", "description": "A possible integer overflow exists in the function `fallback`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/temp/RedLine-0.sol", "function": "fallback", "lineno": 135, "title": "Integer Overflow ", "type": "Warning"}, {"address": 2636, "code": "nt(msg.sender, _stage, now);\n  }\n\n  function findReferre", "debug": "storage_keccac_0: 0x83ffb3569638cea3622d597040d78d7ab7ae5c8a2d8bbb012cad3fb961bad806\nstorage_6: 0xe6089257c80ac0cbe60d5926ff262844c7005edbab82a162281118017d351c97\nstorage_0 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\ncalldatasize_Redline: 0x3\nstorage_78541660797044910968829902406342334108369226379826116161446442989268089806461: 0x0\ncallvalue: 0x0\n", "description": "A possible integer overflow exists in the function `fallback`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/temp/RedLine-0.sol", "function": "fallback", "lineno": 136, "title": "Integer Overflow ", "type": "Warning"}, {"address": 3477, "code": "nder;\n    users", "debug": "storage_2: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_24465873643947496235832446106509767096567058095563226156125564318740882468607: 0xaa108000002bd3db7704854d60910b506d8900000000000000000000000051ac\nstorage_2 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\nUDiv(storage_keccac_storage_24465873643947496235832446106509767096567058095563226156125564318740882468607,\n_____256): 0x0\nstorage_5: 0x1\ncalldatasize_Redline: 0x3\nstorage_0 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0x0\nstorage_78541660797044910968829902406342334108369226379826116161446442989268089806461: 0x0\ncallvalue: 0x0\n", "description": "A possible integer overflow exists in the function `fallback`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/temp/RedLine-0.sol", "function": "fallback", "lineno": 115, "title": "Integer Overflow ", "type": "Warning"}, {"address": 4799, "code": "   for (uint i = 0; i < 3", "debug": "The exception is triggered under the following conditions:\n\nstorage_24465873643947496235832446106509767096567058095563226156125564318740882468607: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_2: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\ncalldatasize_Redline: 0x3\nstorage_2 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\nUDiv(storage_keccac_storage_24465873643947496235832446106509767096567058095563226156125564318740882468607,\n_____256): 0x0\nstorage_5: 0x0\nstorage_0 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0x0\nstorage_78541660797044910968829902406342334108369226379826116161446442989268089806461: 0x0\ncallvalue: 0x0\n", "description": "A reachable exception (opcode 0xfe) has been detected. This can be caused by type errors, division by zero, out-of-bounds array access, or assert violations. This is acceptable in most situations. Note however that `assert()` should only be used to check invariants. Use `require()` for regular input checking. ", "filename": "/temp/RedLine-0.sol", "function": "fallback", "lineno": 151, "title": "Exception state", "type": "Informational"}, {"address": 4989, "code": "sers[referrals[i]].referr", "debug": "The exception is triggered under the following conditions:\n\nstorage_2 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\nUDiv(storage_keccac_storage_24465873643947496235832446106509767096567058095563226156125564318740882468607,\n_____256): 0x1\nstorage_24465873643947496235832446106509767096567058095563226156125564318740882468607: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_5: 0x1\nstorage_2: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\ncalldatasize_Redline: 0x3\nstorage_0 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0x0\nstorage_78541660797044910968829902406342334108369226379826116161446442989268089806461: 0x0\ncallvalue: 0x0\n", "description": "A reachable exception (opcode 0xfe) has been detected. This can be caused by type errors, division by zero, out-of-bounds array access, or assert violations. This is acceptable in most situations. Note however that `assert()` should only be used to check invariants. Use `require()` for regular input checking. ", "filename": "/temp/RedLine-0.sol", "function": "fallback", "lineno": 152, "title": "Exception state", "type": "Informational"}], "success": true}
